include/swift/SIL/SILCloner.h

```diff
@@ -490,6 +490,8 @@ class SILFunctionCloner : public SILClonerWithScopes<SILFunctionCloner> {
   }
 };
 
+std::unique_ptr<SILModule> cloneModule(SILModule &originalModule);
```

Comment:
Does cloning modify the original `SILModule` or is there some sharing going on (i.e. is this not a deep copy)? I would've expected `originalModule` to be `const SILModule &` and not `SILModule &`.